apps/notification/tests/test_apis.py

```diff
@@ -106,6 +109,36 @@ def create_join_request(self, project, user=None):
         )
         return join_request
 
+    # TODO: To test notification after creation of entry comments
+    def create_project(self):
+        analysis_framework = self.create(AnalysisFramework)
+        return self.create(
+            Project, analysis_framework=analysis_framework,
+            role=self.admin_role
+        )
+
+    # TODO: To test notification after creation of entry comments
+    def create_entry(self, project, **fields):
+        lead = self.create(Lead, project=project)
+        return self.create(
+            Entry, lead=lead, project=lead.project,
+            analysis_framework=lead.project.analysis_framework,
+            **fields
+        )
```

Comment:
This already exists in the Base TestCase.
https://github.com/the-deep/server/blob/3bebe249667406624a6316e77de3bf4dadfc2832/deep/tests/test_case.py#L258-L265
https://github.com/the-deep/server/blob/3bebe249667406624a6316e77de3bf4dadfc2832/deep/tests/test_case.py#L226-L234

Reply:
I've removed them